pkgs/servers/home-assistant/appdaemon.nix

```diff
@@ -5,53 +5,47 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "appdaemon";
-  version = "4.2.1";
-  format = "setuptools";
+  version = "4.4.2";
+  pyproject = true;
 
-  disabled = python3.pythonOlder "3.7";
+  disabled = python3.pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner = "AppDaemon";
     repo = "appdaemon";
     rev = "refs/tags/${version}";
-    hash = "sha256-4sN0optkMmyWb5Cd3F7AhcXYHh7aidJE/bieYMEKgSY=";
+    hash = "sha256-T3InE4J4qYeFJTq6nrW8y5BOA7Z0n3t9eVpl641r/xk=";
   };
 
   postPatch = ''
     # relax dependencies
-    sed -i 's/==/>=/' requirements.txt
+    sed -i 's/~=/>=/' pyproject.toml
```

Comment:
Use `pythonRelaxDepsHook` and set `pythonRelaxDeps = true` instead.

Reply:
On second thought, that is not quite the same. Leaving it to you, to decide whether the semantics change is a good idea here.

Reply:
I wasn't sure either. My guess was that the dependency versions specified using `>=` and `<` rather than `~=` have actual significance.

Reply:
I wasn't sure either. My guess was that the dependency versions specified using `>=` and `<` rather than `~=` have actual significance.